src/raft.rs

```diff
@@ -411,12 +411,16 @@ impl<T: Storage> Raft<T> {
         };
         for p in peers {
             let pr = new_progress(1, r.max_inflight);
-            r.mut_prs().insert_voter(*p, pr);
+            if let Err(e) = r.mut_prs().insert_voter(*p, pr) {
+                println!("{}", e);
```

Comment:
this error is serious and is not expected, I prefer panic direct or use fatal!

Reply:
Thanks for the review !
So it should panic in the raft mod ?